ci-operator/step-registry/cucushift/installer/rehearse/aws/usgov/ipi/private/ep/provision/cucushift-installer-rehearse-aws-usgov-ipi-private-ep-provision-chain.yaml

```diff
@@ -0,0 +1,34 @@
+chain:
+  as: cucushift-installer-rehearse-aws-usgov-ipi-private-ep-provision
+  steps:
+    - ref: ipi-conf
+    - ref: ipi-conf-telemetry
+    - ref: aws-provision-vpc-shared
+    - chain: aws-provision-bastionhost
+    - ref: ipi-conf-aws-custom-vpc
+    - ref: ipi-conf-aws
+    - ref: ipi-conf-aws-custom-endpoints
+    - ref: ipi-install-monitoringpvc
+    - ref: proxy-config-generate
+    - chain: ipi-install
+    - ref: aws-provision-tags-for-byo-vpc
+    - ref: enable-qe-catalogsource
+    - chain: cucushift-installer-check
+  env:
+  - name: SERVICE_ENDPOINT_S3
+    default: "s3-fips"
```

Comment:
This endpoint has to work with fips-enabled cluster, right?

Reply:
I don't think so, per my understanding, if enable this endpoint, AWS will do FIPS-validated for data sent to it.